data_models/recipe.py

```diff
@@ -0,0 +1,21 @@
+from dataclasses import dataclass
+from uuid import UUID
+from item import Item
+from enum import Enum
+
+@dataclass
+class recipe:
+    """Class representing meal recipe"""
+    
+    recipe_id: UUID
+    recipe_name: str
+    recipe_style: Enum # appetizer, entree, dessert, etc.
+    recipe_cuisine: Enum
+    recipe_description: str
+    recipe_instructions: dict[int, str]
+    recipe_ingredients: dict[Item, float]
```

Comment:
Hmm maybe instead of float we want to use the `Quantity` enum instead. It might also make sense to remove quantity from the item data model in that case.